config.template.yml

```diff
@@ -313,6 +313,9 @@ authentication_backend:
     ## The attribute holding the display name of the user. This will be used to greet an authenticated user.
     # display_name_attribute: displayName
 
+    ## Follow referrals returned by the server. This is especially useful for environments were read-only servers exist.
```

Comment:
Should be "where" instead of "were". 

Also useful for multi environments where some ldap severs manage specific sub ou's. 

Ive seen companies where the ou=engineering,dc=example,dc=com is referred off to an ldap server the engineering team manage.

Reply:
Do you happen to utilize the OU on a specific server model by chance? I've not yet implemented it for searches, it would be good to see how the library handles responses when some users have referrals and some do not rather than just guessing.